doc/source/ray-air/preprocessors.rst

```diff
@@ -0,0 +1,175 @@
+.. _air-preprocessors:
+
+Preprocessing Data
+==================
+
+This page describes how to perform data preprocessing in Ray AIR.
+
+Data preprocessing is a common technique for transforming raw data into features that will be input to a machine learning model.
+In general, you may want to apply the same preprocessing logic to your offline training data and online inference data.
+Ray AIR provides several common preprocessors out of the box as well as interfaces that enable you to define your own custom logic.
+
+Overview
+--------
+
+Ray AIR exposes a ``Preprocessor`` class for preprocessing. The Preprocessor has 4 methods that make up its core interface.
+
+#. ``fit()``: Compute some state for a Dataset and save it to the Preprocessor.
```

Comment:
This is difficult to understand. Maybe say  "Fit a Preprocessor to a given Dataset." and then give an example of e.g. the mean and std of a scaler that needs to be computed as an example of state to be fitted. "some state" seems too abstract.

Reply:
Yeah... I didn't want to reuse the word "fit" because I felt it doesn't really help explain what it's doing. Took a stab at updating it to be more concrete, let me know what you think.